client/src/App.tsx

```diff
@@ -16,7 +19,9 @@ function App() {
               <h1>Sleep Buddies</h1>
           </header>
           <Routes>
-              <Route path="/" Component={Home}/>
+              <Route path="/" Component={welcome}/>
```

Comment:
hm if we want to do a `remember me` checkbox then we shouldn't have the welcome page as the default page